lib/strapi.js

```diff
@@ -0,0 +1,12 @@
+export function getStrapiURL(path = "") {
+  return `${
+    process.env.NEXT_PUBLIC_STRAPI_API_URL || "http://localhost:1337"
+  }${path}`;
+}
+
+export async function fetchStrapi(path) {
+  const requestUrl = getStrapiURL(path);
+  const response = await fetch(requestUrl);
+  const data = await response.json();
+  return data;
```

Comment:
We'll eventually want some error handling around this, but for now it should be alright.